DQM/python/dqm.py

```diff
@@ -998,21 +1040,43 @@ def __init__(self, name='SampleValidation') :
         self.sim_particles_passname = ''
         self.target_scoring_plane_passname = ''
 
-        # primary histograms
-        self.build1DHistogram("primaries_pdgid", "ID of primary particles", 20, 0, 20)
+        pdgid_bin_labels = [
+            "e^{+}",                   # 1
```

Comment:
Update comment with correct index - current indices are above the correct index by 1.